Devscord.DiscordFramework/Framework/Architecture/Controllers/DiscordCommand.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Devscord.DiscordFramework.Framework.Architecture.Controllers
 {
-    [AttributeUsage(AttributeTargets.Method)]
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
```

Comment:
czy to przypadkiem nie jest domyślne?

Reply:
nie jest domyślne, przynajmniej tak mi się wydaje, bo bez tego był error, że nie attribute nie może być kilka razy użyty